packages/hasura-storage-js/src/hasura-storage-client.ts

```diff
@@ -85,19 +84,10 @@ export class HasuraStorageClient {
       formData = params.formData
     }
 
-    const { fileMetadata, error } = await this.api.upload({
+    return await this.api.upload({
```

Comment:
We are not wrapping things in a try-catch here, so `await` on `return` is [unnecessary](https://eslint.org/docs/latest/rules/no-return-await).

```suggestion
    return this.api.upload({
```